### ISSUE TITLE
Rf/t is not working on screens. 

### ISSUE THREAD

**Issue by mentimjojo** [McJty/RFTools#1960]
## Issue description: 
The Energy module for the screen shows the amount of RF in an item. But when changed to RF/t that flows threw that item doesnt work. It stays on 0RF/t. I tested this on multiple items from different mods. So its a problem within RFtools.

## Steps to reproduce: 
Install RFtools, and for example Flux networks. Use the energy module (Or plus) on a flux point or plug. Now put the module into a screen and change it to rf/t. It will stay on 0. In my case threw this flux point runs 60mil rf/t.

## Versions:
* Minecraft: 1.12.2
* Forge: 1.12.2-14.23.5.2836-universal
* McJtyLib: 1.12-3.5.3
* RFTools: 1.12-7.31

Not using optifine or sponge.

**Comment by McJty** [comment]
The RF/t module measures amount of RF that changes in the block. For example, if you have a plug that has 100 RF/t going in and at the same time 100 RF/t going out then the module will register 0. That's because the power API does not have a way to get this information

**Comment by mentimjojo** [comment]
Damn, quick anwser! Does the power API have methods to get info about incoming and outgoing RF? Ifso you could calculate the RF/t (Yes, I'm a developer to). Or i could write it as an patch and create a PR.

**Comment by McJty** [comment]
No, that info is simply not present. The only information that is available in the API is the amount of power that is contained in the block/item/whatever. It doesn't keep track of how much was inserted since last tick and how much was extracted. And patching this is not going to be easy as this is part of Forge and it would break compatibility with basically all mods that use RF/FE
